transformations/us_state_abbreviation_transformation/transformation.py

```diff
@@ -0,0 +1,145 @@
+import re
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+class USStateAbbreviation(SentenceOperation):
+    tasks = [
+        TaskType.TEXT_CLASSIFICATION,
+        TaskType.TEXT_TO_TEXT_GENERATION,
+    ]
+    languages = ["en"]
+
+    def __init__(self):
+        super().__init__()
+        self.abbreviations = {
+            "Alabama": "AL",
```

Comment:
might make sense to keep them as a separate json file so it's easily expandable without having to modify python files.